pkg/common/test/mock/network.go

```diff
@@ -143,6 +142,7 @@ func (m *SlowReadConn) Peek(i int) ([]byte, error) {
 		time.Sleep(100 * time.Millisecond)
 	}
 	if err != nil || len(b) != i {
+		time.Sleep(m.readTimeout)
```

Comment:
What  is the purpose of add this line?

Reply:
This is the code incorporated by the develop branch.

Reply:
Check the latest develop branch here: https://github.com/cloudwego/hertz/blob/develop/pkg/common/test/mock/network.go#L146
Doesn't look consistent.

Reply:
Removed. Sorry for my mistake.